src/pages/education.tsx

```diff
@@ -0,0 +1,218 @@
+                            />
+                            <path className="st0" d="M656.52,1177.51l35.26-56.18c2.14-7.32-1.1-15.68-8.44-20.27c-14.11-8.8-26.58,8.87-51.34-7.46	c-20.54-13.54-26.2-41.16-12.66-61.7c13.54-20.54,41.16-26.2,61.7-12.66c22.58,14.89,13.94,32.94,28.53,42.05	c5.36,3.35,15.06,3.37,20.86,1.44l35.57-54.16"
+                            />
+                            </svg>
+                        </div>
+                    </Hero>
+                </Container>
+                <Container>
+                    <SectionFeatures>
+                        <h3>How to enhance the learning process with Gitpod</h3>
+                            <div className="features">
+                                <div className="feature">
+                                    <h4>No setup</h4>
+                                    <p>Don't waste time on tedious setup. With Gitpod developer enviroments are automated, reproducible and versioned.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>Works on every device</h4>
+                                    <p>With Gitpod coding gets accessible for  everyone your students can contribute with a Chromebook or a tablet.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>Enhances the workflow</h4>
+                                    <p>Create customized lessons for your students. Everyone can start working on new asignments within minutes.</p>
+                                </div>
+                            </div>
+                            <div className="features">
+                                <div className="feature">
+                                    <h4>Collaborative</h4>
+                                    <p>Gitpod strengthens collaboration between your students. Code reviews has never been this easy and sharable snapshots foster the team workflow.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>Continous Support</h4>
+                                    <p>Easy maintenance, flexible subscriptions and continous support if needed.</p>
+                                </div>
+                                <div className="feature">
+                                    <h4>High Security</h4>
+                                    <p>Data integrity is of utmost importance. Gitpod has high security standards to keep your students and their data safe.</p>
+                                </div>
+                            </div>
+                    </SectionFeatures>
+                </Container>
+                <HeadingMission>Our Mission is to convey our passion for coding to students by eliminating barriers.</HeadingMission>
```

Comment:
Great mission statement! One small typo:

```suggestion
                <HeadingMission>Our mission is to convey our passion for coding to students by eliminating barriers.</HeadingMission>
```